src/index.ts

```diff
@@ -15,7 +15,7 @@ import {AppConfigParams, ShopifyApp, AppConfigInterface} from './types';
 import {AuthConfigInterface} from './auth/types';
 import {WebhooksConfigInterface} from './webhooks/types';
 import {
-  createAuthenticatedRequest,
+  createValidateSession,
   createCspHeaders,
   createEnsureInstalled,
```

Comment:
this is outside the scope of this PR but is there a need to prefix these with `create`? I get that these are functions returning the middleware function because we need to pass in the api and config arguments to it but I don't think there are any conflicts and I think the word `create` here may be misleading to actually creating an authenticated request or creating a valid session.

Reply:
unless we have name conflicts then I don't see why we can't just drop `create`

Reply:
let's save this for a separate PR but I think having the following is sufficient:

```
validateSession(api, config)(req, res)
```